### DISCUSSION TITLE
Daily 17/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#74] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by rpgrca** [comment]
Ayer hicimos la reunion semanal y arme el mapa del sitio. Hoy voy a hacer un caso de uso y continuar con el back-end, para el front-end tal vez podemos armar el esqueleto de la app para dejar preparado los componentes que habria que llenar.

**Comment by NataliaAlvarezIspc** [comment]
El viernes avancé con Angular y las historias de usuarios. También completé la primera versión de la IEEE830. Ayer tuvimos nuestra reunión semanal, en la que participé. Además estuve acomodando y corrigiendo todas las historias de usuarios que estaban mal redactadas y desordenadas y fueron agregadas a la IEEE830.
Hoy continuaré con el diagrama Entidad-Relación. Haré de nuevo el diagrama relacional y el scrip correspondiente y continuaré revisando toda la documentación necesaria para presentar, espero, antes del viernes.
Dificultades: cansancio físico y mental.

**Comment by Brian-Almada** [comment]
Ayer me tomé el día. Hoy me toca hacer el diagrama de casos de uso. Ahora estoy consumiendo un pequeño tutorial para saber cómo hacerlo y luego voy a practicar con los casos de uso que tenemos en el proyecto. Luego, si me queda algo de tiempo voy a ver si ya puedo usar Angular.

**Reply by NataliaAlvarezIspc** [reply]
Acordate de asignarte la tarea de los casos de uso que vas a hacer. Para no superponerte con otros.

**Comment by BraianTroncoso** [comment]
Ayer me puse un poco con Angular, tengo que serguir viendo un poco mas de teoria porque con lo del ISPC no me quedó claro del todo. Hoy voy a ver un poco mas sobre ello y ver si puedo modificar y solucionar el script de la BDD.
